examples/demo/showcase/main.py

```diff
@@ -75,8 +75,8 @@ def go_screen(self, idx):
         self.root.ids.sm.switch_to(self.load_screen(idx), direction='left')
         self.update_sourcecode()
 
-    def go_higherarchy_previous(self):
-        ahr = self.higherarchy
+    def go_heirarchy_previous(self):
+        ahr = self.heirarchy
```

Comment:
I agree.  I think 30, 78, and 79 were in a single commit.  Just merge that?

On Mon, Jan 26, 2015 at 11:52 AM, dessant <notifications@github.com> wrote:

> In examples/demo/showcase/main.py
> <https://github.com/kivy/kivy/pull/2872#discussion_r23558895>:
>
> > @@ -75,8 +75,8 @@ def go_screen(self, idx):
> >          self.root.ids.sm.switch_to(self.load_screen(idx), direction='left')
> >          self.update_sourcecode()
> >
> > -    def go_higherarchy_previous(self):
> > -        ahr = self.higherarchy
> > +    def go_heirarchy_previous(self):
> > +        ahr = self.heirarchy
>
> Lines 30, 78, 79, the third revision will be the best: hierarchy.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kivy/kivy/pull/2872/files#r23558895>.
>